sdk/compiler/damlc/tests/daml-test-files/DivulgeFetchNodeActors/observerAskedToDivulge.EXPECTED.ledger

```diff
@@ -12,20 +12,23 @@ Transactions:
   #1:0
   │   consumed by: #2:0
   │   referenced by #2:0
-  │   disclosed to (since): 'alice' (1), 'divulgee' (1)
+  │   disclosed to (since): 'alice' (1),
+                            'divulgee' (1)
```

Comment:
The actual output includes a random suffix on the end of `alice` and `divulgee` here, which we strip off before comapring to this expected file. However the longer strings ends up leading to a different whitespace formatting, which we still need to accomodate for here, and in other `EXPECTED.ledger` files.